assets/models/organelles/Cytoplasm.tscn

```diff
@@ -5,13 +5,14 @@
 [ext_resource type="Shader" path="res://shaders/OrganelleTransparent.gdshader" id="3"]
 [ext_resource type="Texture2D" uid="uid://cj40olefxw00h" path="res://assets/textures/cytoplasm.png" id="4"]
 
-[sub_resource type="ShaderMaterial" id="1"]
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_v3ckx"]
 resource_local_to_scene = true
 render_priority = 0
 shader = ExtResource("3")
-shader_parameter/dissolveValue = null
-shader_parameter/fresnelValue = 1.0
-shader_parameter/fresnelActive = false
+shader_parameter/dissolveValue = 0.0
+shader_parameter/dissolveScale = 1.0
+shader_parameter/fresnelValue = 0.8
+shader_parameter/fresnelActive = true
```

Comment:
Is it intentional that this model has fresnel mode active? If I remember right that didn't used to be enabled for anything in the game.

Reply:
That's just the parameter that's been in OrganelleOpaque.gdshader since last year at least. Some of the membranes (including the Ferroplast one) were using it previously.
I put it here just to make it more transparent, but I could just make the texture more transparent instead.

Reply:
Yeah I know the parameter was added ages ago but it was removed from all of the organelles. I suppose if it looks fine and the dissolve effect also looks fine with it on for specific organelles, it can be kept on.

Reply:
Was there a specific reason it was removed before?
Also, I forgot to mention that I altered the calculation for it slightly.
It's now used only by lysosome, Signalling Agent, Vacuole and the endosome.

Reply:
I can't remember / know, but I think it was removed as it looked bad. So it's fine to put it back in if it looks good now.